### ISSUE TITLE
[feature] Update icon of Perceptor tab

### ISSUE THREAD

**Issue by heming6666** [hypertrons/hypertrons-crx#177]
## Type of Issue
_What types of issue have you found with hypertrons-crx?_
_Put an `x` in the boxes that apply_

- [ ] Bug
- [x] Feature Request
- [ ] Performance issue
- [ ] Help Wanted
- [ ] Documention incomplete
- [ ] Test Improvement
- [ ] Raise a Question

## Introduction to the issue

Currently the logo of `Perceptor` tab is the same as `insights` tab.

## Expected result

A distinguishable icon for `Perceptor` tab

## screenshot

![image](https://user-images.githubusercontent.com/26642280/122217865-66586f80-cee0-11eb-8ee5-869ae4cf2739.png)

**Comment by LiuChangFreeman** [comment]
It's an svg:  
```
<svg class="octicon octicon-graph UnderlineNav-octicon d-none d-sm-inline" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M1.5 1.75a.75.75 0 00-1.5 0v12.5c0 .414.336.75.75.75h14.5a.75.75 0 000-1.5H1.5V1.75zm14.28 2.53a.75.75 0 00-1.06-1.06L10 7.94 7.53 5.47a.75.75 0 00-1.06 0L3.22 8.72a.75.75 0 001.06 1.06L7 7.06l2.47 2.47a.75.75 0 001.06 0l5.25-5.25z"></path></svg>
```

**Comment by DantrazTrev** [comment]
![Screenshot from 2021-06-18 02-08-01](https://user-images.githubusercontent.com/43457420/122469159-3e4d3700-cfda-11eb-918a-e7622aab9465.png)

I was able to update the icon but what icon do we want for `Perceptor`?

Possibly a mesh?

**Comment by LiuChangFreeman** [comment]
> ![Screenshot from 2021-06-18 02-08-01](https://user-images.githubusercontent.com/43457420/122469159-3e4d3700-cfda-11eb-918a-e7622aab9465.png)
> 
> I was able to update the icon but what icon do we want for `Perceptor`?
> 
> Possibly a mesh?

Very interesting

**Comment by DantrazTrev** [comment]
fixed by #186
